Automattic/jetpack
# Boost: Update e2e tests to account for lazy loading removal

Jetpack removed lazy loading, so these tests have been trimmed down
because of that.
Since our module is disabled by default and can not be activated,
it shouldn't be activatable.



## Proposed changes:

* Fixes e2e tests related to recent lazy loading updates. Since the module is now visible in the UI but can not be activated, the tests now accommodate for that.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Checkout this version of Boost;
* Run e2e tests - they should all pass (hopefully no tunnel issues).
